app/controllers/food_trucks_controller.rb

```diff
@@ -6,4 +6,26 @@ def index
   def show
     @food_truck = FoodTruck.find(params[:id])
   end
+
+  def new
+    @food_truck = FoodTruck.new
+  end
+
+  def create
+    @food_truck = FoodTruck.new(food_truck_params)
+
+    if @food_truck.save
```

Comment:
Please offer the user some kind of indication that the food truck was saved correctly!